interns-react-weekly-assignments/src/components/Home/Ritika/Week4/utils/validations/validations.ts

```diff
@@ -0,0 +1,12 @@
+import warning from "../constants/constants.json"
+
+export const Validations=(taskName:string,taskDescription:string,taskCompletionDate:string)=>{
```

Comment:
use camelCase and instead if validations please specify what kind of validation the function is doing